TemporaryParser/TPRBComment.class.st

```diff
@@ -28,13 +28,28 @@ Class {
 	#superclass : #TPRBNode,
 	#instVars : [
 		'contents',
-		'start'
+		'start',
+		'stop'
 	],
 	#category : #'TemporaryParser-Nodes'
 }

```

Comment:
Add somewhere in the class comment of this class that the contents contain the comments without the quotes.
Also what happens with nested quotes??